### PR TITLE
robot-framework 4.1.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
robot-framework:
  * Binaries built for a non-native architecture were installed into robot-framework's prefix.
    The offending files are:
      /home/linuxbrew/.linuxbrew/Cellar/robot-framework/4.1.1/libexec/lib/python3.9/site-packages/selenium/webdriver/firefox/x86/x_ignore_nofocus.so	(i386)
```

**Comment by carlocab** [comment]
You can probably delete the `x86` directory containing the file.

File: Formula/robot-framework.rb [hunk 1]
```diff
@@ -97,6 +97,11 @@ class RobotFramework < Formula
 
   def install
     virtualenv_install_with_resources
+
+    # remove non-native binary
+    on_linux do
+      rm libexec/"lib/python3.9/site-packages/selenium/webdriver/firefox/x86/x_ignore_nofocus.so"
+    end
```
Comment (carlocab) [inline Formula/robot-framework.rb:104]:
```suggestion
    (libexec/"lib/python3.9/site-packages/selenium/webdriver/firefox/x86/x_ignore_nofocus.so").unlink if OS.linux?
```
We don't use `on_linux` or `on_macos` in `install` and `test` now.

Comment (chenrui333) [reply Formula/robot-framework.rb]: good to know. 👍

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**